core/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexResponse.java

```diff
@@ -32,28 +33,37 @@
 public class CreateIndexResponse extends AcknowledgedResponse {
 
     private boolean shardsAcked;
+    private String index;
+    private Version minimumVersionWithIndex = Version.V_5_6_0;
```

Comment:
They shouldn't diverge as these will never change other than eventually being removed when we no longer have to be backwards compatible with 5.6.0 (say, 7.0.0). This pattern occurs often enough that the field is not needed to clarify that this was added in 5.6.0 instead the code already speaks for itself in that regard. Finally, the git blame will tell us the commit that added this which will have linkage back to this PR if someone *really* needs to go on a cave expedition through that. 😄